devito/arch/archinfo.py

```diff
@@ -227,8 +227,14 @@ def nvidiasmi_gpu_info(text):
                 gpu_info = {}
                 match = re.match(r'GPU *[0-9]*\: ([\w]*) (.*) \(', line)
                 if match:
-                    gpu_info['architecture'] = match.group(1)
-                    gpu_info['product'] = match.group(2)
+                    if match.group(1) == 'Graphics':
```

Comment:
@FabioLuporini This is for the output you showed me on slack.. let me know what you think

Reply:
Looks good to me